PathPlanning/ReedsSheppPath/reeds_shepp_path_planning.py

```diff
@@ -40,19 +41,13 @@ def plot_arrow(x, y, yaw, length=1.0, width=0.5, fc="r", ec="k"):
         plt.plot(x, y)
 
 
-def mod2pi(x):
+def pi_2_pi(x):
     # Be consistent with fmod in cplusplus here.
-    v = np.mod(x, np.copysign(2.0 * math.pi, x))
-    if v < -math.pi:
-        v += 2.0 * math.pi
-    else:
-        if v > math.pi:
-            v -= 2.0 * math.pi
-    return v
+    return angle_mod(x)
 
```

Comment:
This function is same as `angle_mod(x)`?

Reply:
Yup. This function is supposed to do the same thing. I worked on reeds_shepp_path_planning.py for another PR, so I can confirm. Even if the code seems to be different, it is expected to do the same thing.

I can revert the changes if you still think that we should keep the original code.

Reply:
What do you think this PR change? https://github.com/AtsushiSakai/PythonRobotics/pull/487

Reply:
I went through the PR. I understand the concern raised. Both functions (angle_mod and mod2pi in PR #487) wrap any given number between -pi and pi.

Here is a plot of values returned by both functions:
![angle_mod_comparision](https://github.com/AtsushiSakai/PythonRobotics/assets/66770479/50b047bf-ef39-4ddf-851e-1370ef607e11)
As seen in the plots, both functions "mostly" return same values for a given input. 

However, discrepancy occurs between the functions at "positive odd multiples of pi". For example at 3π, returned values are:
1. angle_mod: -π
2. mod2pi: +π
Same values are observed at other positive odd multiples of pi.

Since, these change in signs mean a difference of forward or reverse driving in Reeds-Sheep Path, I agree with the fact that we should keep the function as it was.

Reply:
I will revert the changes soon and commit.